### PR TITLE
how to organize an event

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,4 +101,10 @@ TODO
 
 ## 采用该实践的项目
 
+如果你比较认同这一份最佳实践，欢迎把下面的 badge 添加到你的项目种：
```
Comment (kang8) [inline README.md:104]: typo 
```suggestion
如果你比较认同这一份最佳实践，欢迎把下面的 badge 添加到你的项目中：
```

File: event.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+为达成特定目标，开源社区、组织、个人往往会采用活动（Event）的形式，将相关人群聚集在一起，借助主题分享、动手实践、成果展示等来增强相互间的联系，亦或是宣传、推广特定信息。
+
+从物理空间上来看，可以将活动划分为两大类：线下活动、在线活动（online）。
```
Comment (kang8) [inline event.md:3]: Suggestion
```suggestion
从物理空间上来看，可以将活动划分为两大类：线下活动、线上活动（online）。
```

File: event.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* 适当的中场休息
+* 选择人数最多的时候（可能会有人提前离场），合影留念
+* 活动结束后，清理垃圾，座位恢复摆放等
+
+> 会议开始前，需要提醒参会者主动将喝完的饮料瓶、吃完的食品包装袋扔到指定垃圾箱，保持会场干净、整洁。
+> 如果条件允许的话，可以提供类似于沙漏、计时器等工具来提示讲师的演讲时长。但这些提示工具需要摆放在讲师正对面。
+> 为了能够持续改进活动体验，在结束后邀请大家填写匿名反馈问卷
+> 避免卷入太多角色、利益方，否则会让活动本身变得不再单纯，进而四不像
+
+### 在线活动
```
Comment (kang8) [inline event.md:62]: If the above agrees to change it to “线上活动”, it needs to be modified synchronously.
```suggestion
### 线上活动
```

File: event.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* 场地引导、参会签到
+* 主持人开场（1 分钟）
+* 活动主体
+* 适当的中场休息
+* 选择人数最多的时候（可能会有人提前离场），合影留念
+* 活动结束后，清理垃圾，座位恢复摆放等
+
+> 会议开始前，需要提醒参会者主动将喝完的饮料瓶、吃完的食品包装袋扔到指定垃圾箱，保持会场干净、整洁。
+> 如果条件允许的话，可以提供类似于沙漏、计时器等工具来提示讲师的演讲时长。但这些提示工具需要摆放在讲师正对面。
+> 为了能够持续改进活动体验，在结束后邀请大家填写匿名反馈问卷
```
Comment (kang8) [inline event.md:59]: Missing end-of-line period
```suggestion
> 为了能够持续改进活动体验，在结束后邀请大家填写匿名反馈问卷。
```

**Review by kang8 — COMMENTED** [review]
Some suggestions and typos.

**Comment by LinuxSuRen** [comment]
非常感谢，目前把思路、想法写下来了，内容结构还需要整理。

**Comment by LinuxSuRen** [comment]
短期估计不会更新这个内容，先合并了。

**Pull request merged**